Apps/W1/Shopify/test/.resources/Staff/ShpfyStaffMembers.txt

```diff
@@ -0,0 +1 @@
+{"data":{"staffMembers":{"edges":[{"node":{"accountType":"REGULAR","active":true,"email":"user1@example.com","exists":true,"firstName":"FirstName1","id":"gid://shopify/StaffMember/1234567890","initials":["F","N"],"isShopOwner":true,"lastName":"LastName1","locale":"en","name":"FirstName1 LastName1","phone":null}},{"node":{"accountType":"REGULAR","active":false,"email":"user2@example.com","exists":true,"firstName":"FirstName2","id":"gid://shopify/StaffMember/1234567891","initials":["F"],"isShopOwner":false,"lastName":"LastName2","locale":"en","name":"FirstName2 LastName2","phone":null}}],"pageInfo":{"hasPreviousPage":false,"hasNextPage":false,"startCursor":"START_CURSOR","endCursor":"END_CURSOR"}}},"extensions":{"cost":{"requestedQueryCost":7,"actualQueryCost":4,"throttleStatus":{"maximumAvailable":2000.0,"currentlyAvailable":1996,"restoreRate":100.0}}}}
```

Comment:
Wouldn't it make or sense when this file has the extension `.json` instead of `.txt`